docs/usage/files.md

```diff
@@ -171,6 +171,30 @@ For large files or in cases where the network connection is less reliable,
 you may want to upload the file in parts.  This allows a single part to fail
 without aborting the entire upload, and failed parts can then be retried.
 
+## Automatic Uploader
+
+The SDK provides a method of automatically handling a chunked upload; simply call [`chunked_upload.start()`][start] with
+the path to the path you wish to upload method once you have constructed an [`upload_session`][upload_session] from the
```

Comment:
`the path to the path you wish to upload method` isn't very clear — maybe something like "the path to the file you wish to upload"?